docs/docs/lib/build-your-own.mdx

```diff
@@ -1149,3 +1149,5 @@ Open a [GitHub issue](https://github.com/growthbook/growthbook/issues) with a li
   - Add 2 new test cases for matching on a `$groups` array attribute
 - **v0.5.2** 2023-10-30
   - Add 3 new test cases for comparison operators to handle more edge cases
+- **v0.5.3** 2023-10-30
```

Comment:
- Addressed the `build-your-own` suggestions
- `cases.json` is tricky. I added a case for using `fallbackAttribute`, but testing the actual bucketing logic is tough since most of it requires consecutive E2E runs to ensure the bucketing persists. Open to suggestions here.